tests/host_verify/host/host.cpp

```diff
@@ -198,6 +198,25 @@ static int _verify_report(
         _read_binary_file(
             endorsements_filename, &endorsements_data, &endorsements_file_size);
 
+        result = oe_verify_remote_report(
+            report_data, report_file_size, endorsements_data, endorsements_file_size, NULL);
+        if (pass)
+            OE_TEST(result == OE_OK);
```

Comment:
nit: If there are braces for the else, prefer adding braces for the if as well.

(if the format-code script prefers this format then ignore me)